app/experimenter/nimbus-ui/README.md

```diff
@@ -51,6 +91,52 @@ yarn test -t="renders as expected"
 
 Refer to Jest's [CLI documentation](https://jestjs.io/docs/en/cli) for more advanced test configuration.
 
+### Components that need a GQL Mock
+
+⚠️ TODO: This section needs to be expanded with mock-able properties once Experimenter queries are added.
```

Comment:
I feel the real value of this test helper can't really be seen until we work on the directory view and need to set the cache (which right now is empty). I think we can do the same thing with `<MockedProvider {...{ mocks }}>` as we do here. Should this say "once we need to read and write from the cache" instead of "once Experimenter queries are added"?

Reply:
Yeah that's cool with me! I'll note that since `MockedCache` takes `mocks`, you could still use it over `MockedProvider` (if for no reason other than to be consistent across tests).